src/vsphere_cpi/lib/cloud/vsphere/nsxt_provider.rb

```diff
@@ -85,6 +85,12 @@ def to_s
     end
   end
 
+  class NSXTSecGrpUpdateError < StandardError
```

Comment:
Ruby generally prefers whole words. This should be `NSXTSecurityGroupUpdateError`. On the other hand if this error is being used for 409/412, maybe we should make it generic? `NSXTOptimisticUpdateError`?